.github/actions/init_build.sh

```diff
@@ -22,3 +22,6 @@ bin/console dependencies install --composer-options="$COMPOSER_ADD_OPTS --prefer
 
 # Compile translation files
 php bin/console locales:compile
+
+# Install Panther driver
+vendor/bin/bdi detect drivers
```

Comment:
This is useless too.
```suggestion
```

Reply:
Are you sure? On my local instance, having chrome drivers installe dsystem side does not mean panther will find them. bdi detect is therefore required